src/librbd/cache/ImageCache.h

```diff
@@ -44,13 +100,18 @@ struct ImageCache {
   /// internal state methods
   virtual void init(Context *on_finish) = 0;
   virtual void shut_down(Context *on_finish) = 0;
+  virtual void get_state(bool &clean, bool &empty, bool &present) = 0;
```

Comment:
Nit: seems pointless and highly specialized to just this base class

Reply:
Removed